web/packages/core/src/ruffle-player.ts

```diff
@@ -136,6 +136,7 @@ export class RufflePlayer extends HTMLElement {
     private options: BaseLoadOptions | null;
     private lastActivePlayingState: boolean;
 
+    private showSwfDownload? = false;
```

Comment:
Please make `showSwfDownload` non-nullable, similar to `hasContextMenu`.

Reply:
This is easiest to do by re-introducing the code from https://github.com/ruffle-rs/ruffle/pull/5602#discussion_r745315158. 

If I remove the question mark from this, I get the error "Type 'boolean | undefined' is not assignable to type 'boolean'.
  Type 'undefined' is not assignable to type 'boolean'"

If I then move to load-options and remove the question mark from showSwfDownload there, I get the issue Spread types may only be created from object types. due to the following lines:

const config: BaseLoadOptions = {
                ...(window.RufflePlayer?.config ?? {}),
                ...this.config,
                ...options,
            };